optuna/cli.py

```diff
@@ -187,6 +192,110 @@ def take_action(self, parsed_args: Namespace) -> Tuple[Tuple, Tuple[Tuple, ...]]
         return self._study_list_header, tuple(rows)
 
 
+def _format_trial_values(
+    record: Dict[Tuple[str, str], Any], columns: List[Tuple[str, str]]
+) -> List:
+    row: List = []
+    for column in columns:
+        value = record.get(column, np.nan)
+        if isinstance(value, (int, float)):
+            row.append(value)
+        elif isinstance(value, datetime.datetime):
+            row.append(value.strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
A detail but how about making this format a module local (similar to `dashboard.py`)? The same format is also `_Studies`.